lms/djangoapps/certificates/views/webview.py

```diff
@@ -72,13 +72,14 @@ def get_certificate_description(mode, certificate_type, platform_name):
     elif mode == 'verified':
         # Translators:  This text describes the 'ID Verified' course certificate type, which is a higher level of
         # verification offered by edX.  This type of verification is useful for professional education/certifications
-        certificate_type_description = _("A {cert_type} certificate signifies that a "
+        # ucsd-customizations: Here I am removing {cert_type} as ucsd is only using verified certificates so we
```

Comment:
```suggestion
        # [UCSD_CUSTOM] [EDS-SOMETHING]: Here I am removing {cert_type} as ucsd is only using verified certificates so we
```
can we do something like this for consistency?

Reply:
done.